specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2025-02-01-preview/diagnostic.json

```diff
@@ -1134,8 +1258,26 @@
           ],
           "x-ms-enum": {
             "name": "FilterType",
```

Comment:
can FilterType be made a common object in a common.json file or even elsewhere in this swagger to reduce the duplicate effort?

Reply:
As of now there is no common.json file in DiagnosticRP or under compute but I agree with the duplication of common object. Will keep this as note and we will work towards making common place for such objects.